llvm/lib/Target/RISCV/RISCVSchedule.td

```diff
@@ -257,6 +257,31 @@ def : ReadAdvance<ReadFSqrt16, 0>;
 } // Unsupported = true
 }
 
+multiclass UnsupportedSchedZfhExceptForZfhmin {
```

Comment:
We don't really like adding unused things to the repo. It's likely to get deleted because it doesn't look used.

Reply:
Thanks for reminding! 
Here I'd like to start the discussion: how should we handle the scheduling model of overlapped extensions like `Zfhmin` and `Zfh`? We know that only `Zfhmin` is mandatory in RVA23.
For current implementation, the scheduling model is not so fine-grained. For example, if we only support `Zfhmin`, then we don't need to define `UnsupportedSchedZfh`, but we still need to set `Unsupported` to true for arithmetic operations. And this is what we want to fix in this PR.